type.go

```diff
@@ -561,6 +648,9 @@ func (t int96Type) EstimateDecodeSize(numValues int, src []byte, enc encoding.En
 
 func (t int96Type) AssignValue(dst reflect.Value, src Value) error {
 	v := src.Int96()
+	if dst.Kind() == reflect.Pointer {
+		dst.Set(reflect.ValueOf(&v))
+	}
```

Comment:
Is the code below supposed to be in a `else` branch? Otherwise it seems like this change would have no effect.

Reply:
You are right, we need to exit if the assignment is done, I added a return statement.